addons/test_sale_product_configurators/static/tests/tours/product_configurator_single_custom_attribute_ui.js

```diff
@@ -19,23 +19,10 @@ registry.category("web_tour.tours").add('sale_product_configurator_single_custom
 }, {
     trigger: 'ul.ui-autocomplete a:contains("Customizable Desk (TEST)")',
 }, {
-    trigger: '.oe_advanced_configurator_modal span:contains("Aluminium")',
-    run: function () {
-        // used to check that the radio is NOT rendered
-        if ($('.oe_advanced_configurator_modal ul[data-attribute_id].d-none input[data-value_name="single product attribute value"]').length === 1) {
-            $('.oe_advanced_configurator_modal').addClass('tour_success');
-        }
-    }
```

Comment:
![image](https://user-images.githubusercontent.com/69820391/231453563-c092081d-85bb-47b8-9f86-1703eacdae40.png)

Reply:
Maybe check how the single attribute is rendered then? :thinking: